src/list-articles.tm

```diff
@@ -12,108 +12,118 @@
-  it is not hard to built it yourself.|Mon Dec \ 7 07:28:49 2020 UTC>
-
-  <notes-entry|plugin-status.en.tm|Status of Plugins for <TeXmacs>|We review
-  the current status and limitations of the various plugins on
-  Windows/Mac/Linux.|Sat Dec \ 5 23:36:31 2020 UTC>
-
-  <notes-entry|comments-in-texmacs.tm|Implementing comments in <TeXmacs>|We
-  discuss the implementation of comments in <TeXmacs> documents. This is a
-  new feature introduced by Joris in <tt|r13254> with some additions in
-  subsequent revisions, the current description is based on revision
-  <tt|r13256>.|Sat Dec \ 5 23:35:41 2020 UTC>
-
-  <notes-entry|modular-scheme-graphics.tm|Modular graphics with
-  <name|Scheme>|(no abstract)|Sat Dec \ 5 23:35:37 2020 UTC>
-
-  <notes-entry|editorial-guidelines.tm|Editorial scope and author
-  guidelines|(no abstract)|Sat Dec \ 5 23:35:37 2020 UTC>
-
-  <notes-entry|build-using-cmake-and-homebrew-on-macos.tm|Build <TeXmacs>
-  using CMake and Homebrew |This article serves as a guide for <TeXmacs>
-  contributors and developers on macOS. We talk about how to build and test
-  GNU <TeXmacs> on macOS using CMake and Homebrew.|Sat Dec \ 5 23:35:37 2020
-  UTC>
-
   <notes-entry|articles-wishes.tm|A wishlist for <TeXmacs> articles|It would
   be useful to have some short articles or tutorials covering the topics
   listed here below. This article collects them in the hope that someone will
   contribute them to the blog. They can also be considered interesting
   exercises to improve one own <TeXmacs> knowledge: learning by teaching. In
   case you manage to cover, even partially, one of the topic, please add a
   remark here to point to relevant material. If the topic is covered, move it
-  to the \PAlready covered\Q section.|Sat Dec \ 5 23:35:37 2020 UTC>
+  to the \PAlready covered\Q section.|Fr 19 Feb 2021 14:25:32 GMT>
+
+  <notes-entry|list-articles.tm|List of all the articles|A list of all the
+  articles in the website, ordered by the most recent modification time.|Fr
+  19 Feb 2021 14:18:17 GMT>
+
+  <notes-entry|macro-editor.tm|The Macro editor and easy macro
+  modification|The Macro editor is a shortcut that allows to examine and
+  modify macro definitions. Macros edited via the Macro editor are saved in
+  the preamble of the document the editor is invoked from.|Fr 19 Feb 2021
+  14:18:01 GMT>
 
   <notes-entry|wishlist.tm|Wishlist|This wishlist collects suggestions for
   improvements and nice ideas for addition to TeXmacs. These should be of
   limited scope and somewhat reasonable technical difficulty. It is mostly
   meant for developers or technically savyy users to collect good ideas. It
   should be clear that the functionality is not already provided in some
   form. This wishlist must not be used for very broad and vague requests
-  (e.g. ``make the program more versatile and powerful").|Sat Dec \ 5
-  23:35:28 2020 UTC>
+  (e.g. ``make the program more versatile and powerful").|Do 18 Feb 2021
+  22:44:37 GMT>
```

Comment:
Same problem with locale here. Also maybe in the Atom feed?

Reply:
> I do not think this is necessary. It is not enough for the style file to be in the same directory as the file?

Will check it.

Reply:
> > I do not think this is necessary. It is not enough for the style file to be in the same directory as the file?
> 
> Will check it.

You are right, it can be in the same directory as the file. In this case one can select the style from "Other style" and type in the name (path is not needed, not even relative). Will write it in the README